pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -88,6 +94,20 @@ func NewManager(policyName PolicyName, cr internalapi.RuntimeService, kletGetter
 	}, nil
 }
 
+func getReserverdCores(nodeStatus v1.NodeStatus) int64 {
+	// Kube/System Reserved = Capacity - Allocatable
+	capacity := nodeStatus.Capacity[v1.ResourceCPU]
+	reserved := capacity.Copy()
+	reserved.Sub(nodeStatus.Allocatable[v1.ResourceCPU])
+	glog.Infof("[cpumanager] %v cores reserved for kube/system and shared pool", reserved.Value())
+	if reserved.IsZero() {
+		glog.Infof("[cpumanager] defaulting to 1 core")
```

Comment:
We are going to need to change this at some point as just unilaterally reserving one core messes with the scheduler i.e. on a 4 core node, the schedule sees nothing wrong with scheduling 4 cores of pinned pods to the node; however, the cpumanager can only accommodate 3.

I'm working on new content for the proposal to cover the setting of a `CPUPressure=true` condition on the node to prevent placement of CPUless pods (Bu pods without CPU requests or BE pods) and eviction of existing CPUless pods.  This lets us deplete the shared pool and the scheduler accounting Just Works.

This is just an "keep in mind", not an objection to this change. Thanks!